### ISSUE TITLE
Vimeo video quality controls

### ISSUE THREAD

**Issue by erikmartus** [cookpete/react-player#1680]
I am working on creating a custom player on top of a Vimeo video. Vimeo allows for setting the playback quality via their [SDK](https://github.com/vimeo/player.js#setqualityquality-string-promisestring-typeerrorerror), but this not exposed in the existing Vimeo player. Is it it possible to use this function with react player?

I have gotten it working to a certain extent by doing `playerRef.current.player.player.callPlayer('setQuality', quality)`, and seen that it reports back that the quality has changed from Vimeo using `playerRef.current.player.player.callPlayer('getQuality')`. I have also seen the video playback quality change when doing this, however, when changing from a lower quality to a higher quality this does not cause the video to pause and buffer properly resulting in the video to stop but the audio to continue playing.

**Comment by vivekvdwivedi** [comment]
I am also exploring the same option. Couldn't find it here.

**Comment by luwes** [comment]
are the videos of a Plus, PRO or Business account?
https://github.com/vimeo/player.js#setqualityquality-string-promisestring-typeerrorerror

**Comment by erikmartus** [comment]
> are the videos of a Plus, PRO or Business account? https://github.com/vimeo/player.js#setqualityquality-string-promisestring-typeerrorerror

@luwes yes we have a Business account. We are able to change the quality as I described above by directly calling the function, but the video is not buffering as expected when the quality changes. The buffering occurs when changing the quality when streaming directly on Vimeo and also in our previous implementation of a custom player using VideoJS. In our initial investigation we did not see any differences in the Vimeo config, but were able to confirm that buffer loader we were seeing was native to Vimeo despite controls being disabled in both configs.
